utils.js

```diff
@@ -32,15 +93,41 @@ async function describeTable({ dynamodb, name }) {
   }
 }
 
-async function updateTable({ dynamodb, name, attributeDefinitions }) {
+async function updateTable({prevTable, dynamodb, name, attributeDefinitions, stream, streamViewType }) {
+  const disableStream = prevTable.streamEnabled && !stream
+  const enableStream = !prevTable.streamEnabled && stream
+
   const res = await dynamodb
     .updateTable({
       TableName: name,
       AttributeDefinitions: attributeDefinitions,
-      BillingMode: 'PAY_PER_REQUEST'
+      BillingMode: 'PAY_PER_REQUEST',
+
+      ...(disableStream && {
+        StreamSpecification: {
+          StreamEnabled: false
+        }
+      }),
+
```

Comment:
Ive added this disableStream flag here because the SDK will error if you try to disable a non existing stream. So this flag insures we only add this to the object if the inputs are disabled a stream that exists.